mindjunkies/forums/migrations/0002_remove_forumtopic_reactions_forumcomment_likes_and_more.py

```
@@ -8,41 +7,33 @@
         ),
         migrations.AddField(
-            model_name="forumtopic",
-            name="likes",
-            field=models.ManyToManyField(
-                related_name="likedTopics",
-                through="forums.LikedPost",
-                to=settings.AUTH_USER_MODEL,
-            ),
+            model_name='forumtopic',
+            name='likes',
+            field=models.ManyToManyField(related_name='likedTopics', through='forums.LikedPost', to=settings.AUTH_USER_MODEL),
```

Ruff (0.8.2)

Line too long (126 > 119)

(E501)